prime-router/metadata/HL7/catchall/hl7/segments/PRT/PRTExtension.yml

```diff
@@ -0,0 +1,293 @@
+# $schema: ./../../../../../json_schema/fhir/hl7-to-fhir-mapping-resource-template.json
+
+url:
+  type: SYSTEM_URL
+  value: prt-extension
+
+extension:
```

Comment:
Should we map PRT.1?

Reply:
I initially attempted it but had problems populating it. I noticed OBXExtension (from which this is patterned) similarly doesn't map OBX.1 so I filed that away to check later.

Reply:
Ah this is correct, just remembered while looking at FHIR->HL7 mappings that we just increment with an index.